### PR TITLE
Templates for interviewer

### PR DESCRIPTION
**Pull request by oleg08**
code review:
- [ ] @CHILL99 
- [x] @DimaSmagulov 
- [ ] @krisberry 
- [x] @natalya5 
- [x] @OrestF 
- [x] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba 

https://trello.com/c/9yrkhYfJ/46-bug-interviewer-should-be-able-to-manage-templates

### CODE REVIEW

**Comment by Kmalynss** [comment]
1. One pull request should include implementation of one task and
  link to this one i found [here](https://trello.com/c/zHWfFy74/43-bug-when-editing-template-and-changing-questions-questions-should-not-be-reset) and [here](https://trello.com/c/9yrkhYfJ/46-bug-interviewer-should-be-able-to-manage-templates). 
2. I can't see how this PR fixes any of linked issues. I see that admin ```before_action :check_admin``` was removed but no other check added and it looks like any one now can who enters proper url can manage templates. Also I couldn't find any change related to question reseting bug.

File: app/views/templates/_all_templates.html.slim [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.questions-left
+= link_to 'All templates', templates_path
```
Comment (OrestF) [inline app/views/templates/_all_templates.html.slim]: I think its not a good idea to make partial with two lines of code!

**Comment by oleg08** [comment]
only authenticated and approved users can access templates
question-reset bug is  not reproducible. It was just a misunderstanding

**Comment by Kmalynss** [comment]
@oleg08 please give me a link to code which prevents unauthenticated user to access templates

**Comment by vvs79** [comment]
reviewed

File: app/views/templates/_template.html.slim [hunk 2]
```diff
@@ -0,0 +1,11 @@
+tr
+  td class="question"
```
Comment (krisberry) [inline app/views/templates/_template.html.slim]: Classes in slim can be wrote like `td.question` but it's not mistake (: Here goes next question, where have you described this class?

Comment (Kmalynss) [inline app/views/templates/_template.html.slim]: @krisberry why did you put a tic if you have questions? You should put tick only if from your perspective everything is ok.

File: app/views/templates/_template.html.slim [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    = template.name
+  td class="question"
```
Comment (Kmalynss) [inline app/views/templates/_template.html.slim]: BTW why did you use ```question``` class name here?

File: app/views/templates/_template.html.slim (hunk 2 above)
Comment (oleg08) [inline app/views/templates/_template.html.slim]: Similar table was in question so I re-used class "question". But now it is excessive, so I removed it

Comment (oleg08) [inline app/views/templates/_template.html.slim]: code which prevents unauthenticated user to access templates:
in controller:
before_action :authenticate_user!

in routes:
devise_scope :user do
    authenticated :user do
      root 'users#show', as: :authenticated_root
    end
  
    unauthenticated do
      root 'devise/sessions#new', as: :unauthenticated_root
    end
  end
I tried to go to the template as unauthenticated and received: get '*a' => 'static_pages#error'

**Comment by natalya5** [comment]
reviewed

**Comment by rtershak** [comment]
reviewed.

File: app/views/templates/_form.html.slim [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .form-group
-  = form_for([:admin, @template]) do |f|
+  = form_for([@template]) do |f|
```
Comment (Kmalynss) [inline app/views/templates/_form.html.slim]: please change ```= form_for([@template]) do |f|``` to ```= form_for(@template) do |f|```

File: app/views/templates/_template.html.slim [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  td
+    = template.name
+  td
+    = link_to 'Show', template_path(template),      role: 'button', class: 'btn btn-info'
+  td
+    = link_to 'Edit', edit_template_path(template), role: 'button', class: 'btn btn-info'
+  td
+    = link_to 'Destroy', template_path(template),   role: 'button', class: 'btn btn-info',
+              method: :delete,
+              data: { confirm: 'Are you sure?' }
```
Comment (Kmalynss) [inline app/views/templates/_template.html.slim]: add new line symbol

**Comment by oleg08** [comment]
addressed suggestions

**Comment by Kmalynss** [comment]
:+1:

**Pull request merged**